wordpress-mobile/WordPress-iOS
# Issue/update mediapicker pod to 0.15

Updates media picker to version 0.15, this version brings the following:

solves some issues with the capturing of images and video on the picker
introduces a control bar to video previews
solves issues with animation/scrolling issues when the picker is started
To test:

Start the app
Create a new post
Tap on the media button
Check if picker shows correctly
Tap on the live capture header
Capture an image or video
Check if all updates correctly
Force touch (long press on non force touch devices), on a video asset and see if the preview shows correctly.
Tap on the video and see if the control bar shows up and you can play and pause
Check if the status bar of the preview has the correct color.

Needs review: @astralbodies
